modules/end-user-guide/partials/proc_configuring_gitlab_authentication.adoc

```diff
@@ -0,0 +1,74 @@
+// configuring-gitlab-authentication
+
+[id="configuring_gitlab_authentication_{context}"]
+= Authenticating on Gitlab servers
+
+Configuring authentication on Gitlab system is a very similar to Bitbucket.
+
+Gitlab authentication is based on using personal access tokens. Each Gitlab user is able to request some
+amount of personal access tokens with different names, permissions, expiration times, and so on. Those tokens
```

Comment:
can we put a link to `GitLab` documentation?

Reply:
\+